docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+wordpress:
+  build: .
+  links:
+    - db:mysql
+  ports:
+    - 8080:80
+  volumes:
+    - ./wp-content/themes/blank:/var/www/html/wp-content/themes/blank
```

Comment:
I don't know why, it took a lot of time for me to found it so I think we must change it.